src/calculation/ClauseResultsHelpers.ts

```diff
@@ -360,7 +360,13 @@ export function isSupplementalDataElementStatement(
 ): boolean {
   if (supplementalDataElements != undefined) {
     for (const supplementalData of supplementalDataElements) {
-      if (supplementalData.criteria.language === 'text/cql' && supplementalData.criteria.expression === statementName) {
+      // text/cql-identifier if correct, but text/cql.identifier used to be correct so for backwards compatibility we want to support both
```

Comment:
```suggestion
      // text/cql-identifier is correct, but text/cql.identifier used to be correct so for backwards compatibility we want to support both
```

I think?

Might also want to link to the conformance requirement in the comment: https://build.fhir.org/ig/HL7/cqf-measures/measure-conformance.html#conformance-requirement-3-7